### PR TITLE
Remove the courtroom

### PR DESCRIPTION
**Pull request by MciReloaded**
## About The Pull Request

Removes the courtroom, which was taking space due to the holodeck having a courtroom option. Replaces it with a public shooting range!
## Why It's Good For The Game

Gives a few chances at RP (shooting competition, anyone?), a chance for characters to practice their aim and shooting, and uses space that was otherwise being wasted.
## Changelog
:cl:
add: added in public shooting range
add: practice round Frozen Star vendor
del: removed the courtroom
/:cl:

## Screenshots
image
Take a look at the new NEV Northern Light Shooting Range! Earmuffs, charging stations, and targets included.

### CODE REVIEW

**Comment by EvilJackCarver** [comment]
Flagging don't-merge: one of the checks is erroring out and I need to look into why and if it can be ignored.

...Sometime after I wake up. Got a lot to do tomorrow.

File: maps/CEVEris/_CEV_Erida.dmm [hunk 1]
```diff
@@ -79687,6 +79630,16 @@
...
+	prices = list(/obj/item/ammo_magazine/ammobox/lrifle_small/practice=20,/obj/item/ammo_magazine/ammobox/shotgun/practiceshells=20,/obj/item/ammo_magazine/ammobox/pistol/practice=20,/obj/item/ammo_magazine/ammobox/magnum/practice=20,/obj/item/ammo_magazine/ammobox/pistol/practice=20,/obj/item/ammo_magazine/ammobox/clrifle_small/practice=20,/obj/item/ammo_magazine/ammobox/pistol/practice=20,/obj/item/ammo_magazine/ammobox/srifle_small/practice=20);
+	products = list(/obj/item/ammo_magazine/ammobox/lrifle_small/practice=200,/obj/item/ammo_magazine/ammobox/shotgun/practiceshells=200,/obj/item/ammo_magazine/ammobox/pistol/practice=200,/obj/item/ammo_magazine/ammobox/magnum/practice=200,/obj/item/ammo_magazine/ammobox/pistol/practice=200,/obj/item/ammo_magazine/ammobox/clrifle_small/practice=200,/obj/item/ammo_magazine/ammobox/pistol/practice=200,/obj/item/ammo_magazine/ammobox/srifle_small/practice=200);
+	product_slogans = "The best offense is practice!;Buy for your whole family today!;Nobody can outsmart bullet!;God created man - Frozen Star made them EQUAL!;Stupidity can be cured! By LEAD.;Dead kids can't bully your children!;Fix that shitty aim!"
```
[the remark this answers is not in the document]
Comment (MciReloaded) [reply maps/CEVEris/_CEV_Erida.dmm:79639]: I. Yeah I guess so. I did it all by hand in SDMM to be fair, I was unaware of the define method.

**Comment by MciReloaded** [comment]
tl;dr it's the practice laser gun causing issues with trying to use a cell. May have messed up the define on them

**Comment by MciReloaded** [comment]
The error reads as follows,
```
[10:28:02] Runtime in energy.dm,119: Cannot read /obj/item/cell/medium/high (/obj/item/cell/medium/high).charge
  proc name: update icon (/obj/item/gun/energy/update_icon)
  src: the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice)
  src.loc: the floor (84,132,3) (/turf/simulated/floor/tiled/dark/gray_platform)
  call stack:
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): update icon(null)
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): refresh upgrades()
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): refresh upgrades()
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): set firemode(1)
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): Initialize(1)
  the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice): Initialize(1)
  Atoms (/datum/controller/subsystem/atoms): InitAtom(the NT LR M \"Lightfall\" - P (/obj/item/gun/energy/laser/practice), 0, /list (/list))
  Atoms (/datum/controller/subsystem/atoms): InitializeAtoms(null, null)
  Atoms (/datum/controller/subsystem/atoms): Initialize(376756)
  Master (/datum/controller/master): Initialize(10, 0, null)

```
Three times

File: maps/CEVEris/_CEV_Erida.dmm (hunk 1 above)
[the remark this answers is not in the document]
Comment (EvilJackCarver) [reply maps/CEVEris/_CEV_Erida.dmm]: The define method is... literally how every item in the game exists.

If you're looking for the code, it's code/game/machinery/vending or something to that effect.

[the remark this answers is not in the document]
Comment (MciReloaded) [reply maps/CEVEris/_CEV_Erida.dmm]: I have no idea why I said I was unaware. It was late and I thought the mapping method would be easier.



I was wrong.

[the remark this answers is not in the document]
Comment (EvilJackCarver) [reply maps/CEVEris/_CEV_Erida.dmm]: Just wanted to touch base on the status of this?
